support/packages/github-to-jira-synchronization-tool/src/constants.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * SPDX-FileCopyrightText: © 2020 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: MIT
+ */
+
+const PASSWORD = process.env.PASSWORD;
+const PORT = process.env.PORT || 5000;
+const PROJECT = process.env.PROJECT || 'IFI';
+const SECRET = process.env.SECRET;
+const USERNAME = process.env.USERNAME;
+
+module.exports = {
```

Comment:
I like hiding this into a module, instead of placing `process.env.xxx`s all over the code :+1: